### PR TITLE
feat: productPage and refactor code

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
added:
1) Product Page - it consist of sidebar of  the product page .
2) refactored previous codes .

### CODE REVIEW

**Comment by SuyashTiwariCoder** [comment]
image

File: src/Layouts/ProductsPage/ProductsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "../../Layouts/ProductsPage/ProductPage.css";
+import {
+  ProductSidebar,
+  ProductMainContent,
+} from "../AllComponents/AllComponents";
+
+function ProductsPage() {
+  return (
+    <>
```
Comment (fahd23) [inline src/Layouts/ProductsPage/ProductsPage.jsx:10]: You can remove <>, as div is the only element to be return

**Review by fahd23 — APPROVED** [review]
Given suggestion

File: src/Shared/SidebarContainerCard/SidebarContainerCard.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+function SidebarContainerCard({ data }) {
+  return (
+    <>
+      <div className="sidebar--container">
+        <div className="sidebar--heading">{data[0].heading}</div>
+        <div className="sidebar--content">
+          <label htmlFor={data[0].type1}>
+            <input type="checkbox" id={data[0].type1} />
+            &nbsp;&nbsp;&nbsp;{data[0].input1}
```
Comment (munjalSaksham71) [inline src/Shared/SidebarContainerCard/SidebarContainerCard.jsx:11]: ```&nbsp;``` is. a non-line breaker and provide you empty space. You need to use only one in a line code.

**Review by munjalSaksham71 — APPROVED** [review]
Code Looks Fine. Just one suggestion as mentioned below.

File: src/Shared/SidebarContainerCard/SidebarContainerCard.jsx (hunk 2 above)
Comment (SuyashTiwariCoder) [reply src/Shared/SidebarContainerCard/SidebarContainerCard.jsx:11]: ok

File: src/Layouts/ProductsPage/ProductsPage.jsx (hunk 1 above)
Comment (SuyashTiwariCoder) [reply src/Layouts/ProductsPage/ProductsPage.jsx:10]: ya sure

**Pull request merged**